DuckDuckGo/BrowserChromeManager.swift

```diff
@@ -19,15 +19,143 @@
 
 import UIKit
 
+private class BarsAnimator {
```

Comment:
Maybe move this to the bottom?  I personally feel we should try and keep the main class of the file as near to the top as possible if we can.